src/hooks/gqty.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'lib/gqty';
+
+import { useSelectedCircle } from 'recoilState';
+
+export const useCurrentOrg = () => {
+  const { circleId } = useSelectedCircle();
+  const query = useQuery();
+  const currentOrg = query
+    .organizations({ where: { circles: { id: { _eq: circleId } } } })
+    .map(org => ({
+      id: org.id,
```

Comment:
I don't think this is needed, to separate out each, because the whole thing that gqty is doing is only querying what you use

Reply:
the hook here is intended to return an actual query result, not a live query that the hook consumers would add their own fields to. it may not look like the "ideal usage" of gqty but i'm trying to find something that gets us moving in the right direction in the context of what we've built already. definitely open to revisiting all this later